lib/_stream_writable.js

```diff
@@ -108,6 +108,31 @@ function WritableState(options, stream) {
 
   // True if the error was already emitted and should not be thrown again
   this.errorEmitted = false;
+
+  // count buffered requests
+  this.bufferedRequests = 0;
+
+  // the requests that needs to be called by uncork
+  this.corkedCbs = null;
+
+  // call all the corked requests
+  this.afterCorkedWrite = function afterCorkedWrite(err) {
+    var state = stream._writableState;
+    var entry = state.corkedCbs;
+    var cbs = entry.cbs;
+
+    state.corkedCbs = entry.next;
```

Comment:
Has this property already been set on the object? If not I would suggest placing it, even just `this.corkedCbs = null;` in the constructor. This will help prevent the object map from changing, and aide the optimizing compiler.